libs/ui/editor/src/lib/hooks/index.ts

```diff
@@ -0,0 +1 @@
+export * from './unsavedChangesDialog'
```

Comment:
Just be aware that barrel files are very slow for bundlers, jest etc (learned this a few weeks ago) to parse which may cause issues in the future. But no need to care for now.

I might check what the impact for us is but I don't think it's too big due to using nx.